src/backend/TrafficCourts/Common/Features/Mail/Model/MailTemplateCollection.cs

```diff
@@ -45,18 +45,9 @@ public class MailTemplateCollection
             {
                 TemplateName = "VerificationEmailTemplate",
                 Sender = "DoNotReply@tickets.gov.bc.ca",
-                SubjectTemplate = "Verify your email for traffic violation ticket <ticketid>.",
-                HtmlContentTemplate = "In order to confirm submission of your intent to dispute traffic violation ticket <ticketid> click on the following link." +
-                    "<br/><br/><a href='http://<baseref>/email?uuid=<emailverificationtoken>'>http://<baseref>/email?uuid=<emailverificationtoken></a>" +
-                    "<br/><br/>If you need more help, contact the Violation Ticket Centre toll free 1-877-661-8026, open weekdays 9am to 4pm."
-            },
-        new MailTemplate()
-            {
-                TemplateName = "ResendEmailVerificationTemplate",
-                Sender = "DoNotReply@tickets.gov.bc.ca",
-                SubjectTemplate = "Verify your email for traffic violation ticket <ticketid>.",
-                HtmlContentTemplate = "In order to confirm submission of your intent to dispute traffic violation ticket <ticketid> click on the following link." +
-                    "<br/><br/><a href='http://<baseref>/verifiyEmail?uuid=<emailverificationtoken>'>http://<baseref>/email?uuid=<emailverificationtoken></a>" +
+                SubjectTemplate = "Verify your email for traffic violation ticket {ticketid}.",
+                HtmlContentTemplate = "In order to confirm submission of your intent to dispute traffic violation ticket {ticketid} click on the following link." +
```

Comment:
Just wanted to verify that this still works using {} instead of <>

Reply:
Yes, tested. {} instead of <> works. I did this because <> is less confusing with HTML markup in the same string.